multiqc/modules/pbmarkdup/pbmarkdup.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from pbmarkdup"""
+
+import logging
+import re
+
+from collections import OrderedDict
+
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+    pbmarkdup module class, parses pbmarkdup output.
+    """
+
+    def __init__(self):
+        super(MultiqcModule, self).__init__(
+            name="pbmarkdup",
+            anchor="pbmarkdup",
+            href="https://github.com/PacificBiosciences/pbmarkdup",
+            info=(
+                """
+                 takes one or multiple sequencing chips of an amplified libray
+                 as HiFi reads and marks or removes duplicates.
+                 """
+            ),
+            # Can't find a DOI // doi=
+        )
+
+        self.pbmarkdup = dict()
+
+        for logfile in self.find_log_files("pbmarkdup", filehandles=True):
+            self.pbmarkdup[logfile["s_name"]] = self.parse_logfile(logfile)
+            self.add_data_source(logfile)
```

Comment:
If we have a `return False` in the code for dodgy files (see comment below):

```suggestion
            data = self.parse_logfile(logfile)
            if data:
                self.pbmarkdup[logfile["s_name"]] = data
                self.add_data_source(logfile)
```

Reply:
Added